benchexec/tools/sv_benchmarks_util.py

```diff
@@ -40,3 +44,196 @@ def get_data_model_from_task(task, param_dict):
                     f"Unsupported data_model '{data_model}' defined for task '{task}'"
                 )
     return None
+
+
+def _partition_input_files(input_files, task_options):
+    """
+    This function partitions the input files into witness and other files.
+    The distinction is based on the file name of the witness file, which
+    is identified by the option "witness" in the task options.
+
+    @param input_files: List of input files
+    @param task_options: Dictionary of task options
+    @return: Tuple of witness files and other files
+    """
+    witness_files = []
+    other_files = []
+    for file in input_files:
+        if Path(file).name == task_options.get("witness"):
+            witness_files.append(file)
+        else:
+            other_files.append(file)
+    return witness_files, other_files
+
+
+def get_witness(task):
+    """
+    This function returns the unique witness file from the task.
```

Comment:
Please remove all occurrences of "unique" unless you are actually talking about uniqueness, e.g., when eliminating duplicates.

Reply:
Sorry forgot about fixing this in the documentation, improved in 267f9266f09fdcf2c480f07606020d827efb49ab